libcodechecker/analyze/analyzers/analyzer_clangsa.py

```diff
@@ -172,9 +172,18 @@ def construct_analyzer_cmd(self, result_handler):
                                          '-Xclang', checker_name])
 
             if config.ctu_dir and not self.__disable_ctu:
+                # ctu-clang5 compatibility
                 analyzer_cmd.extend(['-Xclang', '-analyzer-config',
                                      '-Xclang',
-                                     'xtu-dir=' + self.get_xtu_dir()])
+                                     'xtu-dir=' + self.get_ctu_dir()])
+                # ctu-clang6 compatibility (5.0 and 6.0 options work together)
+                analyzer_cmd.extend(['-Xclang', '-analyzer-config',
+                                     '-Xclang',
+                                     'experimental-enable-naive-ctu-analysis'
+                                     '=true',
```

Comment:
I do not like this line break before `=true`. Maybe starting the list in the next line would help to reduce the indent? This way we could keep this string together.

Reply:
pep8, flake8 and pylint liked this.

Reply:
Agreed, this is very bad for the eye. You could try the following:

```python
analyzer_cmd.extend(
    ['-Xclang', '-analyzer-config',
     '-Xclang', 'experimental-enable-naive-ctu-analysis=true'])
```

Or if it doesn't work, you could also create a string above with this literal. If everything else fails, as a module "constant".